modules/appeals_api/spec/docs/v2/sc_spec.rb

```diff
@@ -159,6 +159,189 @@
+      let(:'X-VA-Birth-Date') { '1900-01-01' }
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_file_number_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:veteran_insurance_policy_number_header]
+
+      if DocHelpers.wip_doc_enabled?(:sc_v2_claimant)
+        parameter AppealsApi::SwaggerSharedComponents.header_params[:claimant_first_name_header]
+        parameter AppealsApi::SwaggerSharedComponents.header_params[:claimant_last_name_header]
+      end
+
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:consumer_username_header]
+      parameter AppealsApi::SwaggerSharedComponents.header_params[:consumer_id_header]
+
+      response '200', 'Valid Minimum' do
+        let(:sc_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_200995.json')))
+        end
+
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'sc_validate.json')))
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
+
+        it 'minimum fields used' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
+      end
+
+      response '200', 'Valid maximum' do
+        let(:sc_body) do
+          JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'fixtures', 'v2', 'valid_200995_extra.json'))).tap do |data|
+            unless DocHelpers.wip_doc_enabled?(:sc_v2_claimant)
+              data.dig('data', 'attributes')&.delete('claimant')
+              data.dig('data', 'attributes')&.delete('claimantTypeOtherValue')
+              data['data']['attributes']['claimantType'] = 'veteran'
+            end
+          end
+        end
+
+        if DocHelpers.wip_doc_enabled?(:sc_v2_claimant)
+          let(:'X-VA-Claimant-First-Name') { 'first' }
+          let(:'X-VA-Claimant-Last-Name') { 'last' }
+        end
+
+        schema JSON.parse(File.read(AppealsApi::Engine.root.join('spec', 'support', 'schemas', 'sc_validate.json')))
+
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          response_title = example.metadata[:description]
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              examples: {
+                "#{response_title}": {
+                  value: JSON.parse(response.body, symbolize_names: true)
+                }
+              }
+            }
+          }
+        end
```

Comment:
Did the `rswag example` shared spec not work for these examples? I thought I'd built in something that could account for the `examples: {}` response.

Reply:
I'll look. The error sections were being really difficult